software/02-OpenBCI_Streaming_Data_Format.md

```diff
@@ -54,36 +62,78 @@ Note: values are 24-bit signed, MSB first
 * Bytes 21-23: Data value for EEG channel 6  
 * Bytes 24-26: Data value for EEG channel 8  
 
-**Accelerometer Data**  
-Note: values are 16-bit signed, MSB first
-
-* Bytes 27-28: Data value for accelerometer channel X  
-* Bytes 29-30: Data value for accelerometer channel Y  
-* Bytes 31-32: Data value for accelerometer channel Z  
+**Aux Data**
+* Bytes 27-32: 6 bytes of data defined and parsed based on the **Footer** below
 
 **Footer**  
 
-* Byte 33: 0xC0
+* Byte 33: 0xCX where X is 0-F in hex
+
+#### Firmware Version 1 (2014 to Fall 2016)
+
+Stop Byte | Byte 27 | Byte 28 | Byte 29 | Byte 30 | Byte 31 | Byte 32
```

Comment:
Ah this table is sorted by `Stop Byte`. This is a table which indicates how to parse the `aux` bytes based on the `Stop Byte`